bootstrap.sh

```
@@ -1,4 +1,25 @@
-#!/usr/bin/env bash
+k#!/usr/bin/env bash
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)